spec/components/requestBodies/Organization.yaml

```diff
@@ -4,13 +4,50 @@ content:
       required:
         - name
         - website
+        - country
```

Comment:
I think that field should not be required

Reply:
what about reporting currency?  I imagine we default to USD.

Reply:
@adamaltman yep 
> Default: "USD"
>The currency used for reports. It can only be set upon organization creation.

https://rebilly.github.io/RebillyReportsAPI/#tag/Organizations

should it be a requited ?